### ISSUE TITLE
Vortex claims it "failed to delete savegame(s)"

### ISSUE THREAD

**Issue by Steamphunky** [Nexus-Mods/Vortex#12881]
**Describe the bug** 
When I attempt to delete a save file it gets deleted but a red error message pops up, the save game gets deleted and even booting vortex back up does not restore it.
Error: TypeError: Cannot read properties of undefined (reading 'refreshImportSaves') C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\gamebryo-savegame-management\index.js:1:7255

**To Reproduce**
I have no idea, i don't know what caused the error sadly

**Expected behavior**
Save game is deleted, no error claiming to have a permissions problem.

**Screenshots**

**Platform (please complete the following information):**
 - OS: Windows 10
 - Vortex Version  1.6.7 (the most recent one if this is outdated, it just updated the other day, before this started happening)

**Additional context**
Context: the past few days anytime I make a save it gets corrupted within minutes, I've done a fresh install of everything I can think of. When that did not help, I googled the problem and got no results that helped me find out what the problem is, and nothing. I tried to hunt down the file that it claims to not have access to "index.js:1:7255" and there's an "index" file but not "index.js:1:7255". I've never had a problem like this before and I don't remember doing anything that would cause the missing file to get deleted, and I'm at a loss as to how I should proceed. 
Side note: I accidentally hit the enter button on my number pad and comment this before i had even begun to write anything.

**Comment by Steamphunky** [comment]
> A clear and concise description of what the bug is
Error: TypeError: Cannot read properties of undefined (reading 'refreshImportSaves') C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\gamebryo-savegame-management\index.js:1:7255
Context: the past few days anytime I make a save it gets corrupted within minutes, I've done a fresh install of everything I can think of. When that did not help, I googled the problem and got no results that helped me find out what the problem is, and nothing. I tried to hunt down the file that it claims to not have access to "index.js:1:7255" and there's an "index" file but not "index.js:1:7255". I've never had a problem like this before and I don't remember doing anything that would cause the missing file to get deleted, and I'm at a loss as to how I should proceed.
